scripts/dependency_install_functions.sh

```diff
@@ -49,6 +49,16 @@ else
   fi
 fi
 
+if [[ ! -v PKG_CONFIG_PATH ]]; then
```

Comment:
What is this variable used for? PKG_CONFIG_PATH this could benefit from a comment.